### ISSUE TITLE
[AI Brief] AMS: Detecting Unsafe and Tampered Language Models via Activation Analysis

### ISSUE THREAD

**Issue by toshiOnlii** [toshiOnlii/tech-docs#51]
Source: Google Research Publications (publication)
Source ID: google-research-pubs
URL: https://research.google/pubs/ams-detecting-unsafe-and-tampered-language-models-via-activation-analysis/
Published: unknown
Relevance score: 0.5
Relevance reason: generic technical relevance; google keyword hits=language,model

## Research summary
AMS: Detecting Unsafe and Tampered Language Models via Activation Analysis Glen Messenger Zenodo (2026) Download Google Scholar Copy Bibtex Abstract We introduce AMS (Activation-based Model Scanner), a tool for verifying whether a language model is safe to deploy by analyzing its internal activation patterns. While "uncensored" and maliciously fine-tuned models pose increasing risks, current detection methods rely on behavioral testing that is slow, incomplete, and easily evaded. AMS takes a fundamentally different approach: measuring the geometric structure of safety-relevant concepts in the model's activation space. Safe models exhibit strong class separation (4-8σ) between harmful and benign content; models with removed or degraded safety training show collapsed separation (<2σ). Using contrastive prompt pairs and direction vector analysis, AMS performs model-level verification rather than prompt-level classification. We validate AMS across 14 model configurations spanning 3 architecture families (Llama, Gemma, Qwen), 3 quantization levels (FP16, INT8, INT4), and multiple model categories (instruction-tuned, base, abliterated, uncensored). In our validation set: (1) all four instruction-tuned models pass with 3.8-8.4σ separation; (2) three tested uncensored models (Dolphin, Lexi, LLama-3-8b-Uncensored) flagged as CRITICAL with 1.1-1.3σ on harmful content; (3) an abliterated Llama variant flagged as WARNING (3.33σ); (4) Llama base model shows 0.69σ, confirming absence of safety training; (5) quantization has minimal impact (<5% drift). One model labeled "uncensored" (Dar…

Tags: research, ml, publication
Latest run: 2026-06-13T00-00-52+00-00

**Comment by toshiOnlii** [comment]
## Writer draft ready
Slug: `ams-detecting-unsafe-and-tampered-language-models`

- AMS（Activation-based Model Scanner）は、言語モデル内部の活性化空間における有害／無害概念の分離度（σ）を測り、安全性や改変の有無を判定する手法を提案する。
- 対照的なプロンプトペアと方向ベクトル解析により、振る舞いテストではなくモデルレベルでの検証を行い、4–8σの分離を「安全」、2σ未満の崩壊を「不十分」とみなす。
- Llama・Gemma・Qwenの14構成で検証し、uncensored系（Dolphin等）を1.1–1.3σで危険判定、量子化（FP16/INT8/INT4）は分離度への影響が小さい（<5%）と報告する。

**Comment by toshiOnlii** [comment]
## QC result: PASS
Score: 0.875

- missing or failed: has_source_grounded_implications
- 実装・運用への示唆が一般論に寄りすぎ。本文に即した判断材料へ寄せる

**Comment by toshiOnlii** [comment]
## Published
Slug: `ams-detecting-unsafe-and-tampered-language-models`
Output: /home/toshi/.openclaw/workspace/ai-tech-brief-business/content/2026/05/agentic-ai-infrastructure-in-practice-learn-these.md

**Comment by toshiOnlii** [comment]
## Writer draft ready
Slug: `ams-detecting-unsafe-and-tampered-language-models`

- AMS（Activation-based Model Scanner）は、言語モデルの内部活性化の幾何構造を用いて安全性や改変の有無を判定する手法を提案する。
- 有害/無害プロンプトの対照ペアから方向ベクトルを抽出し、活性空間でのクラス分離度（σ）を測定することでモデル単位の安全性を評価する。
- 振る舞いベースのテストに比べ回避耐性と網羅性の向上が期待され、改変・検閲除去モデルの検出に新しい基準を与える点が重要である。

**Comment by toshiOnlii** [comment]
## QC result: PASS
Score: 0.875

- missing or failed: has_source_grounded_implications
- 実装・運用への示唆が一般論に寄りすぎ。本文に即した判断材料へ寄せる

**Comment by toshiOnlii** [comment]
## Published
Slug: `ams-detecting-unsafe-and-tampered-language-models`
Output: {"ok": true, "branch": "main", "file": "/home/toshi/Project/tech-docs/articles/agentic-ai-infrastructure-in-practice-learn-these.md", "repo": "toshiOnlii/tech-docs"}

**Comment by toshiOnlii** [comment]
## Writer draft ready
Slug: `ams-detecting-unsafe-and-tampered-language-models`

- AMS（Activation-based Model Scanner）は、言語モデル内部の活性化パターンから安全性を検証する手法として提案された。
- 有害／無害コンテンツに対する表現の分離度（σ）を測定し、コントラストプロンプトと方向ベクトル解析でモデル単位の健全性を判定する。
- 従来の挙動テストに比べて回避耐性と網羅性が期待され、改変モデルや安全性劣化の検出に新しい基準を与える可能性がある。

**Comment by toshiOnlii** [comment]
## QC result: PASS
Score: 1.0

- meets current automated checks

**Comment by toshiOnlii** [comment]
## Published
Slug: `ams-detecting-unsafe-and-tampered-language-models`
Output: {"ok": true, "branch": "main", "file": "/home/toshi/Project/tech-docs/articles/agentic-ai-infrastructure-in-practice-learn-these.md", "repo": "toshiOnlii/tech-docs"}
